mu4e/mu4e-compose.el

```diff
@@ -913,13 +919,19 @@ beginning of the buffer."
 (define-key mu4e-compose-mode-map
   (vector 'remap 'beginning-of-buffer) 'mu4e-compose-goto-top)
 
-(defun mu4e-compose-goto-bottom ()
+(defun mu4e-compose-goto-bottom (&optional arg)
   "Go to the end of the message or buffer.
 Go to the end of the message (before signature) or, if already there, go to the
-end of the buffer."
-  (interactive)
+end of the buffer.
+
+Push mark at previous position, unless either a \\[universal-argument] prefix
+is supplied, or Transient Mark mode is enabled and the mark is active."
+  (interactive "P")
+  (or arg
+      (region-active-p)
+      (push-mark))
   (let ((old-position (point))
-   (message-position (save-excursion (message-goto-body) (point))))
+	(message-position (save-excursion (message-goto-body) (point))))
```

Comment:
No reason to change the indentation here.

Reply:
Emacs did it by itself and I let it because it's right here. I can revert if you want.

Reply:
@djcb has the final word on that.  I mentioned it justbecause it is unrelated to the purpose of this change.